### PR TITLE
Add feature freeze note

### PR DESCRIPTION
**Pull request by BigBlueHat**
Solves for https://github.com/w3c/json-ld-wg/issues/84

Let me know if this is copacetic and I'll do the same for framing and syntax.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,13 @@
 
 This is the repository of the W3C’s specification on JSON-LD 1.1 Processing Algorithms and API, developed by the [JSON-LD Working Group](https://www.w3.org/2018/json-ld-wg/). The editors’ draft of the specification can also be [read directly](https://w3c.github.io/json-ld-api/).
 
+## Feature Frozen
+
+We are no longer accepting new features for JSON-LD 1.1 drafts. Please limit
```
Comment (iherman) [inline README.md]: features -> feature requests

File: README.md [hunk 2]
```diff
@@ -5,6 +5,13 @@
...
+requests to bugs and clarifications of the existing feature set.
```
Comment (iherman) [inline README.md]: requests -> issues

File: README.md (hunk 1 above)
Comment (BigBlueHat) [reply README.md]: fixed in latest.

File: README.md (hunk 2 above)
Comment (BigBlueHat) [reply README.md]: fixed in latest, but with a slightly more verbose text to avoid the hard to read "issues to bugs" combo.

**Pull request merged**